src/BloomExe/CollectionTab/LibraryListView.cs

```diff
@@ -759,13 +782,6 @@ private void SelectBook(BookInfo bookInfo)
 
 		private Book.Book SelectedBook
 		{
-			set
```

Comment:
Doesn't seem related to task...maybe you just found this to be redundant when dealing with everything that used the Tag?